framework/include/interfaces/Coupleable.h

```diff
@@ -1436,8 +1442,10 @@ class Coupleable
   /// Whether the MooseObject is a finite volume object
   const bool _is_fv;
 
-private:
   const MooseObject * const _obj;
```

Comment:
you're not using this so why change it to protected

Reply:
This is not changing it to protected. There is already a `private` designation a few lines above